Python/pystate.c

```diff
@@ -1031,18 +1031,6 @@ _PyInterpreterState_SetNotRunningMain(PyInterpreterState *interp)
 {
     PyThreadState *tstate = interp->threads.main;
     assert(tstate == current_fast_get());
```

Comment:
To fix the unused variable warning:

```suggestion
    assert(interp->threads.main == current_fast_get());
```